otkafka/setup_test.go

```diff
@@ -9,15 +10,21 @@ import (
 )
 
 func TestMain(m *testing.M) {
+	if os.Getenv("KAFKA_ADDR") == "" {
+		fmt.Println("Set env KAFKA_ADDR to run otkafka tests")
+		os.Exit(0)
+	}
+
 	setupTopic()
+
 	os.Exit(m.Run())
 }
 
 func setupTopic() {
 	var topics = []string{"trace", "test", "example"}
 
 	for _, topic := range topics {
-		conn, err := kafka.Dial("tcp", "localhost:9092")
+		conn, err := kafka.Dial("tcp", os.Getenv("KAFKA_ADDR"))
```

Comment:
If it's cluster. is that ok ?

Reply:
The plan of this PR is to support env for testing. Maybe cluster support and full feature env controls can be added in seprate PRs

Reply:
ok. GOOD WORK.